utils/build/docker/java/spring-boot-native.Dockerfile

```diff
@@ -0,0 +1,45 @@
+
+FROM eclipse-temurin:8 as agent
+
+# Install required bsdtar
+RUN apt-get update && \
+	apt-get install -y libarchive-tools
+# Install tracer
+COPY ./utils/build/docker/java/install_ddtrace.sh binaries* /binaries/
+RUN /binaries/install_ddtrace.sh
+
+
+FROM ghcr.io/graalvm/graalvm-ce:ol8-java11-22 as build
+
+# Install maven
+RUN curl https://archive.apache.org/dist/maven/maven-3/3.8.6/binaries/apache-maven-3.8.6-bin.tar.gz --output /opt/maven.tar.gz && \
+	tar xzvf /opt/maven.tar.gz --directory /opt && \
+	rm /opt/maven.tar.gz
+
+WORKDIR /app
+
+# Copy application sources
+COPY ./utils/build/docker/java/spring-boot/pom.xml .
+COPY ./utils/build/docker/java/spring-boot/src ./src
+RUN mv ./src/main/resources/application-native.properties ./src/main/resources/application.properties
+
+# Copy tracer
+COPY --from=agent /dd-tracer/dd-java-agent.jar .
+
+# Build native application
+RUN /opt/apache-maven-3.8.6/bin/mvn package -P spring-native
+
+FROM adoptopenjdk:11-jre-hotspot
```

Comment:
Is hotspot required? What about using only `ubuntu` to ensure Java is not used / required?